package.json

```diff
@@ -14,49 +14,57 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
     "ember-classic-decorator": "^2.0.0",
-    "ember-cli-babel": "^7.21.0",
-    "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli-babel": "^7.23.0",
+    "ember-cli-htmlbars": "^5.3.1",
```

Comment:
We are making a downgrade, we can undo the change for this line `ember-cli-htmlbars`.